packages/entity-list/src/modules/list/sagas.js

```diff
@@ -328,12 +328,15 @@ function* getPreferencesSorting() {
 export const FALLBACK_SORTING_FIELD = 'update_timestamp'
 export function* setSorting() {
   const {formDefinition, entityModel} = yield select(listSelector)
+  const {sorting: inputSorting} = yield select(inputSelector)
   const tableSorting = yield call(getSorting, formDefinition)
   const preferencesSorting = yield call(getPreferencesSorting)
   const activeSearchFilterHasOrderBy = yield call(hasActiveSearchFilterOrderBy)
   if (preferencesSorting && preferencesSorting.length > 0) {
     yield put(actions.setSorting(preferencesSorting))
-  } else if (tableSorting.length > 0) {
+  } else if (inputSorting && inputSorting.length > 0) {
```

Comment:
@PadPalon Can't this be shortened?
`inputSorting?.length > 0`

Reply:
Right, I'm not used to optional being available in JS yet :D